frontend/packages/console-shared/src/components/drawer/__tests__/Drawer.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Drawer from '../Drawer';
+import { DraggableCore } from 'react-draggable';
+import { CSSTransition } from 'react-transition-group';
+
+describe('DrawerComponent', () => {
+  it('should exist', () => {
+    const wrapper = shallow(<Drawer />);
+    expect(wrapper.isEmptyRender()).toBe(false);
+  });
+
+  it('should have default values', () => {
+    const wrapper = shallow(
+      <Drawer>
+        <p>This is content</p>
+      </Drawer>,
+    );
+    expect(wrapper.find(DraggableCore).exists()).toBe(false);
+    const style = wrapper.find('.ocs-drawer').prop('style');
+    expect(style.height).toEqual(300);
+    expect(wrapper.find('p')).toHaveLength(1);
```

Comment:
Okay, probably wasn't really critical of this... I think it might be worth adding a dummy class name (something like `test-class-x` so it can't be confused) to the `p` in your example... if you ever use a `p` in Drawer, you run the risk of this test failing.